be/functions/src/services/termsService.js

```
@@ -15,21 +15,99 @@ class TermsService {
-    return this.updateUserTerms(uid, termsData);
+  /**
+   * 약관 데이터를 Firestore 업데이트 객체로 변환
+   * @param {Object} termsData
+   * @param {string} uid
+   * @return {Object} Firestore 업데이트 객체
+   */
+  prepareTermsUpdate(termsData, uid) {
+    const {serviceVersion, privacyVersion, age14Agreed, pushAgreed, termsAgreedAt} = termsData;
+    
+    const termsUpdate = {};
+    if (serviceVersion) termsUpdate.serviceTermsVersion = serviceVersion;
```

Biome (2.1.2)

[error] Duplicate class member name "validateTermsData"



(lint/suspicious/noDuplicateClassMembers)